candybot/candybot/engine/candy.py

```diff
@@ -91,7 +91,8 @@ def list_str(self):
 
     @property
     def line_str(self):
-        return ", ".join(x.small_str for x in self._items if x.value is not 0)
+        sortlist = sorted(self._items, key=lambda x: x.value, reverse=True)
```

Comment:
Can you rename the variable something like `sorted_candy`?
Other then that, looks ok!